js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/invertedIndex/InvertedIndexFieldsDataEntry.tsx

```diff
@@ -84,7 +84,9 @@ const SelectedFieldDetails = ({
               ...invertedIndexFieldsMap.fields,
               isRequired: false,
               isDisabled: !window.frontendConfig.isEnterprise,
-              tooltip: "Nested fields are available on Enterprise plans",
+              tooltip: window.frontendConfig.isEnterprise
+                ? undefined
+                : "Nested fields are available on Enterprise plans.",
```

Comment:
This wording only works for ArangoGraph but not on-prem. Can we just say "available in the Enterprise Edition"?